### PR TITLE
Terraform archive

### LINKED ISSUE #62: Terraform Archive: lambda and event bridge
### Task:
- [x] Terraform lambda function 
- [x] Terraform event bridge and link it to lambda function

### User story
As a developer I want to create lambda and event bridge as infrastructure as code, allowing for repeatability and easy maintainability of my resources.

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
terraform lambda for archiving 
terraform event bridge to call lambda everyday once

closes #62

### CODE REVIEW

File: archive/DOCKERFILE [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+RUN pip install -r requirements.txt
+
+COPY db_connection.py .
+COPY clean_national_rail.py .
+COPY clean_real_time_trains.py .
+COPY archive.py .
+
+CMD [ "archive.handler" ]
+
+# docker build --platform "linux/amd64" -t c11-railway-tracker-archive-lambda-ecr .
```
Comment (fm1psy) [inline archive/DOCKERFILE]: This comment should not be part of the final build, as this is just a personal note

File: archive/clean_national_rail.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    """ Cleans NationalRail Incidents from the RDS based on whether the incident has ended 
+        or not."""
+
+    conn = db_connection.get_connection()
+    delete_old_incidents(conn)
+    conn.close()
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO,
```
Comment (fm1psy) [inline archive/clean_national_rail.py]: Good logging config

**Review by fm1psy — DISMISSED** [review]
This looks good, great job

File: archive/DOCKERFILE (hunk 1 above)
Comment (FarihaChoudhury) [reply archive/DOCKERFILE]: agreed!

File: archive/requirements.txt
Comment (A-Faris) [inline archive/requirements.txt:1]: Nice spelling

**Review by A-Faris — APPROVED** [review]
Looks good

**Pull request merged**